packages/components/src/components/linear-progress/linear-progress-container.jsx

```diff
@@ -2,61 +2,51 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { LinearProgress } from './linear-progress.jsx';
 
-class LinearProgressContainer extends React.Component {
-    constructor(props) {
-        super(props);
-        this.state = {
-            timeout: props.timeout / 1000,
-            total_ticks: Math.round(props.timeout / 1000),
-            current_tick: Math.round(props.timeout / 1000),
-        };
-    }
+const LinearProgressContainer = ({ timeout, action, render, ...other_props }) => {
```

Comment:
Could we grab `className` here and remove `other_props`?

Reply:
Yes, let's do it since it doesn't need any other properties